packages/amplify-e2e-core/src/init/amplifyMock.ts

```diff
@@ -0,0 +1,14 @@
+import { ExecutionContext, getCLIPath, nspawn as spawn } from '..';
```

Comment:
if `amplify mock` is top level command then should this file go to `packages/amplify-e2e-core/src/categories/mock.ts`

Reply:
I'd say no, I don't consider mock to be a category, it's more of a subcommand applying to many categories. So when you run a generic mock, it's mocking all the other categories without specifically targeting one